code/src/components/TodoList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import todos from '../reducers/todos';
+
+const TodoList = () => {
+  const items = useSelector((store) => store.todos.items);
+  const dispatch = useDispatch();
+
+  return (
+    <div>
+      {items.map((todo) => (
+        <div key={todo.id} className="todo-item">
+          <button type="button" className="delete-btn" onClick={() => dispatch(todos.actions.removeTodo(todo.id))}>
+            X
+          </button>
+          <p>{todo.description}</p>
+          <input
+            type="checkbox"
+            checked={todo.isComplete}
+            onChange={() => dispatch(todos.actions.toggleComplete(todo.id))} />
```

Comment:
IMHO, Layout: checkbox, text, delete btn, wóuld be more intuitive. 
To make it even cleaner you can wrap checkbox and text with a container and kick your delete button by using `space-between`